test/unit/logic/transactionPool.js

```diff
@@ -0,0 +1,138 @@
+var TransactionPool  = require('../../../logic/transactionPool.js');
+var TransactionLogic = require('../../../logic/transaction.js');
+var DelegateModule   = require('../../../modules/delegates.js');
+
+var TransferLogic = require('../../../logic/transfer.js');
+
+var transactionTypes = require('../../../helpers/transactionTypes');
+var AccountModule    = require('../../../modules/accounts.js');
+var BlocksModule     = require('../../../modules/blocks.js');
+var AccountLogic     = require('../../../logic/account.js');
+var Rounds           = require('../../../modules/rounds.js');
+var modulesLoader    = require('../../common/initModule').modulesLoader;
+var async            = require('async');
+var expect           = require('chai').expect;
+var node             = require('../../node');
+var ws               = require('../../common/wsCommunication');
+
+function postTransaction(transaction, done) {
+  ws.call('postTransactions', {
+    transaction: transaction
+  }, done, true);
+}
+
+describe('transactionPool', function () {
+  let txPool;
```

Comment:
We are still on `es5`. The project is scheduled to be updated at the end of the current milestone.